Library/Homebrew/cask/blacklist.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Cask
+  module Blacklist
+    def self.blacklisted_reason(name)
+      case name
+      when /^adobe\-.+/i
+        "Adobe Casks are not actively maintained by Homebrew."
+      when /audacity/i
+        "Audacity is not actively maintained by Homebrew."
```

Comment:
`"Audacity was removed because it is too difficult to download programmatically."`